documents/vci-flow.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+
+header VCI flow
+title
+VCI flow
+A scenario for getting a credential in your SSI wallet
+end title
+
+autonumber
+
+participant "User" as user order 0 #ORANGE
+participant "Wallet" as wallet order 1 #RED
+participant "VCI front end" as vci order 2 #BLUE
+participant "Agent" as agent order 3 #GREEN
+
+user -> vci: from the list, select a credential that you want to receive
+vci -> agent: fetch credential metadata/requested presentation definition\nand generate a qr code
+vci -> user: show the qr code
+user -> wallet: scan the qr code
+wallet -> wallet: search the credentials inside the wallet\nfor matching with the presentation definition
```

Comment:
Whut? That has 0 todo with VCI!

Reply:
Yeah, I was going for showing the complete thing oid4vp+oid4vci. removed the oid4vp part from this file